### PR TITLE
add dataSet actions in project page

### PR DESCRIPTION
**Pull request by eperedo**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/8696rgk6z

### :video_camera: Screenshots/Screen capture

image

### :fire: Notes to the tester

#8696rgk6z

### CODE REVIEW

**Review by MiquelAdell — APPROVED** [review]
I tested the changes and they work.

Earlier I did merge a minor PR because it did not need further review and I think this may have caused that this PR now has some conflicts. Could you please fix them whenever you have the time?

Thanks!

**Comment by eperedo** [comment]
> I tested the changes and they work.
> 
> Earlier I did merge a minor PR because it did not need further review and I think this may have caused that this PR now has some conflicts. Could you please fix them whenever you have the time?
> 
> Thanks!

Done @MiquelAdell

File: src/webapp/components/dataset-table/DataSetActions.tsx
Comment (tokland) [inline src/webapp/components/dataset-table/DataSetActions.tsx:1]: [global] after a "yarn localize" I see changes for the i18n files.

File: src/webapp/components/projects/ProjectTable.tsx
Comment (tokland) [inline src/webapp/components/projects/ProjectTable.tsx:148]: minor, no need to change it, but as a general rule, naked boolean arguments are not declarative when used: `reloadProjects(false)`. Instead: `reloadProjects({ isCancelAction: false })`

**Review by tokland — APPROVED** [review]
All good, just two minor comments that need not my further approval, so approving now.

**Pull request merged**
